LuaUI/Widgets/cmd_airpad_excluder.lua

```diff
@@ -0,0 +1,27 @@
+-------------------------------------------------------------------------------------
+-------------------------------------------------------------------------------------
+
+function widget:GetInfo()
+    return {
+      name      = "Airpad Excluder",
+      desc      = "Handler for exclude airpads gadget.",
+      author    = "SharkGameDev",
+      date      = "2020-09-26",
+      license   = "PD",
+      handler   = true,
+      layer     = 0,
+      enabled   = true,
+    }
+end
+
+VFS.Include("LuaRules/Configs/customcmds.h.lua")
```

Comment:
Nit: no need to pollute the global namespace with a ton of `CMD_FOO` constants from the `.h` version, use `customcmds.lua`:
```suggestion
local CMD_EXCLUDEAIRPAD = VFS.Include("LuaRules/Configs/customcmds.lua").EXCLUDEAIRPAD
```

Reply:
Widget has been removed entirely in latest commit.